tajo-docs/src/main/sphinx/functions/hivefunc.rst

```diff
@@ -0,0 +1,81 @@
+##############
+Hive Functions
+##############
+
+Tajo provides a feature to use Hive functions directly without re-compilation or additional code.
+
+*************
+Configuration
+*************
+
+Only thing to do is registering path to a directory for jar files containing your hive functions.
+You can do this by set ``tajo.function.hive.code-dir`` in ``tajo-site.xml`` like the following.
+
+.. code-block:: xml
+
+  <property>
+    <name>tajo.function.hive.code-dir</name>
+    <value>/path/to/hive/function/jar</value>
+  </property>
+
+.. note::
+  The path should be one in local filesystem. HDFS directory is not supported because of JAVA URI compatability problem.
+
+.. warning::
+
+  The path must point to a directory, not a file. And multiple directory entries are not allowed.
+  However, it is possible to load multiple jar files.
+
+***************
+Using in detail
+***************
+
+=============
+Function Name
+=============
+
+Tajo reads hive functions override ``org.apache.hadoop.hive.ql.exec.UDF`` class. Function name is used as specified in
+``@Description`` annotation. If it doesn't exist, Tajo uses full qualified class name as function name. For example,
+it can be like this : ``select com_example_hive_udf_myupper('abcd')``, so it is recommended to use Description annotation.
+
+And if some function conflict occurs, it may throw ``AmbiguousFunctionException``. This conflict means about function signature,
+not only about function name.
+
+============================
+Parameter type / Return type
+============================
+
+Hive uses *Writable* type of Hadoop in functions, but Tajo uses its internal *Datum* type.
+Because Tajo doesn't support a kind of pluggable type system yet, only some Writable types are supported currently by internal converting.
```

Comment:
I think the statement 'Because Tajo doesn't support a kind of pluggable type system yet' is unnecessary. IMO, it may cause other misunderstanding.